jmeter-service/jmeterUtils.go

```diff
@@ -210,9 +214,9 @@ func executeJMeter(testInfo TestInfo, workload *Workload, resultsDir string, url
 	if result && err != nil {
 		logger.Debugf("Successfully executed JMeter test: %v", testInfo)
 	} else {
-		logger.Errorf("Successfully executed JMeter test: %v", testInfo)
+		logger.Errorf("Could not parse JMeter test (%v). Error: %s  ", testInfo, err.Error())
 	}
-	return result, err
+	return result, nil
```

Comment:
Why do you return `nil` here? If an error actually happened, shouldn't it also be returned?
And if no error happened, `err` is `nil` anyways.

Reply:
at that point in code err is nill but if you want I can substitute it with err

Reply:
But you could have an error, print the error message and then still return `nil` instead of an error object. Couldn't that be a problem?